client/idm/console/src/main/java/org/apache/syncope/client/console/topology/Topology.java

```diff
@@ -486,18 +486,19 @@ protected void onTimer(final AjaxRequestTarget target) {
 
                 target.appendJavaScript("checkConnection()");
 
-                if (getUpdateInterval().seconds() < 5.0) {
-                    setUpdateInterval(Duration.seconds(5));
-                } else if (getUpdateInterval().seconds() < 10.0) {
-                    setUpdateInterval(Duration.seconds(10));
-                } else if (getUpdateInterval().seconds() < 15.0) {
-                    setUpdateInterval(Duration.seconds(15));
-                } else if (getUpdateInterval().seconds() < 20.0) {
-                    setUpdateInterval(Duration.seconds(20));
-                } else if (getUpdateInterval().seconds() < 30.0) {
-                    setUpdateInterval(Duration.seconds(30));
-                } else if (getUpdateInterval().seconds() < 60.0) {
-                    setUpdateInterval(Duration.seconds(60));
+                long seconds = getUpdateInterval().getSeconds();
+                if (seconds < 5.0) {
+                    setUpdateInterval(java.time.Duration.ofSeconds(5));
```

Comment:
Because of the `import` above, please replace `java.time.Duration` with `Duration`, thanks.